03.asynchronous/callback_non_error.js

```diff
@@ -0,0 +1,29 @@
+import sqlite3 from "sqlite3";
+
+const db = new sqlite3.Database(":memory:", () => {
+  db.run(
+    "CREATE TABLE books(id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, title VARCHAR(20) UNIQUE NOT NULL)",
```

Comment:
他の DBMS のような VARCHAR 型は SQLite にはないので、最大長を指定しても意味がないと思います。SQLite のドキュメントを確認して、適切なデータ型を使用してください。また、INSERT 文とスタイルを揃えるなら、以下のようにしたほうがよいと思います。

```suggestion
    "CREATE TABLE books (id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, title VARCHAR(20) UNIQUE NOT NULL)",
```